docsite-astro/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "open-props",
+  "author": "Adam Argyle",
+  "browserslist": [
+    "defaults"
+  ],
+  "scripts": {
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro build",
+    "preview": "astro preview"
+  },
+  "devDependencies": {
+    "astro": "^0.24.3"
```

lets wait until we can release with v1 of astro, hopefully soon!
```suggestion
    "astro": "1.0.0"
```